src/content/docs/en/recipes/docker.mdx

```diff
@@ -128,6 +128,38 @@ http {
 }
 ```
 
+#### Multi-stage build
+Here's an example of a more advanced Dockerfile that, thanks to Docker's [multi-stage builds](https://docs.docker.com/build/building/multi-stage/), optimizes the build process for your site, sometimes reducing the build time even by minutes.
```

Comment:
I think this title might be better, since we are showing an example using Node adapter
I also combined both alineas into one, which makes it a bit shorter!

```suggestion
### Multi-stage build (using SSR)

Here's an example of a more advanced Dockerfile that, thanks to Docker's [multi-stage builds](https://docs.docker.com/build/building/multi-stage/), optimizes the build process for your site by only reinstalling the npm dependencies when the source code changes. This can reduce the build time even by minutes, depending on the size of your dependencies.
```